src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -534,6 +534,23 @@ public function resolveCurrentStatement(Node $node): ?Stmt
         return null;
     }
 
+    private function resolveNeigborStmt(StmtsAwareInterface $stmtsAware, Stmt $node, int $key, bool $isPrevious = true): ?Node
```

Comment:
I think its written "neighbor"

Reply:
not sure about this `isPrevious` flag.. maybe it should be 2 methods instead?

Reply:
I found that the issue is only on `next` node, I rename to `resolveNeighborNextStmt()` only for only to cover next node https://github.com/rectorphp/rector-src/pull/3887/commits/83c1b59ff3c78782816b6b873ee29893d67cfefe